bin/sqlline-thin.py

```diff
@@ -18,64 +18,58 @@
 # limitations under the License.
 #
 ############################################################################
-
+from __future__ import print_function
 import os
+import phoenix_utils
 import subprocess
 import sys
-import phoenix_utils
-import atexit
 import urlparse
 
-global childProc
-childProc = None
-def kill_child():
-    if childProc is not None:
-        childProc.terminate()
-        childProc.kill()
-        if os.name != 'nt':
-            os.system("reset")
-atexit.register(kill_child)
```

Comment:
AFAICT, that variable was not altered anywhere after the initialisation. I grepped across the project (ugh.. globals) and did not see it changed it anywhere else either. 

IMO, this is a copy-paste artifact?